recipes/finetuning_openai_summaries.ipynb

```diff
@@ -0,0 +1,1723 @@
+{
```

Comment:
<p>we should explain that the other summaries are generated using a temperature of 1 to have some randomness, whereas the first generation has a temperature of 0</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>